modules/ROOT/pages/assembly_tasks.adoc

```diff
@@ -0,0 +1,14 @@
+[]
+= Tasks
+
+
+A `Task` or a `ClusterTask` is a collection of sequential steps you would want to run as part of your continuous integration flow. A task will run inside a pod on your cluster.
```

Comment:
- @joaedwar: Are these commands that a user runs? "Running" does not sound like an appropriate word to use along with steps to me.  How about rephrasing as something like:

"A `Task` or a `ClusterTask` is a collection of sequential steps you complete as part of your continuous integration flow."

- Also, do not use the future tense in technical documents if you can use the present tense to convey the same information.
Here, "A task runs inside a pod on your cluster" reads good enough. What do you think?